src/main/scala/com/monsanto/arch/cloudformation/model/resource/ApiGateway.scala

```diff
@@ -0,0 +1,260 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model.{ConditionRef, ResourceRef, Token, `Fn::GetAtt`}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+import DefaultJsonProtocol._
+
+case class `AWS::ApiGateway::Account`(
+                                       name: String,
+                                       CloudWatchRoleArn: Option[String] = None,
```

Comment:
I'm guessing this and a few other things should likely be `Option[Token[_]]` since they would like come in either as parameters or resource refs.